ChangeFeedLabSolution/ChangeFeedProcessor.cs

```diff
@@ -50,25 +51,20 @@ public static class ChangeFeedProcessor
             string eventHubNamespaceConnection = ConfigurationSettings.AppSettings["EventHubNamespaceConnection"];
 #pragma warning restore CS0618 // Type or member is obsolete
 
-            // Build connection string to access event hub within event hub namespace.
-            EventHubsConnectionStringBuilder eventHubConnectionStringBuilder =
-                new EventHubsConnectionStringBuilder(eventHubNamespaceConnection)
-                {
-                    EntityPath = EventHubName
-                };
-
-            // Create event hub client to send change feed events to event hub.
-            EventHubClient eventHubClient = EventHubClient.CreateFromConnectionString(eventHubConnectionStringBuilder.ToString());
-
-            // Iterate through modified documents from change feed.
-            foreach (var doc in documents)
+            // Create producer client to send change feed events to event hub.
+            await using (var producer = new EventHubProducerClient(eventHubNamespaceConnection, EventHubName))
             {
-                // Convert documents to json.
-                string json = JsonConvert.SerializeObject(doc);
-                EventData data = new EventData(Encoding.UTF8.GetBytes(json));
-
-                // Use Event Hub client to send the change events to event hub.
-                eventHubClient.SendAsync(data);
+                using EventDataBatch eventBatch = await producer.CreateBatchAsync();
+                // Iterate through modified documents from change feed.
+                foreach (var doc in documents)
+                {
+                    // Convert documents to json.
+                    string json = JsonConvert.SerializeObject(doc);

```

Comment:
We should update this to use `System.Text.Json` and remove the `Newtonsoft` reference.